Components/Services/Installments/InstallmentsPaymentBuilderService.php

```diff
@@ -42,6 +65,20 @@ public function getPayment(PaymentBuilderParameters $params)
      */
     private function getReturnUrl()
     {
+
+        if (version_compare($this->dependencyProvider->getVersion(),'5.6.0','>=')) {
```

Comment:
Normally is a version check bad practice, maybe consider feature detection over it :)

Reply:
Any hint on how to do this?